openmls/src/group/mod.rs

```diff
@@ -80,22 +80,22 @@ impl GroupContext {
 
 /// Configuration for an MLS group.
 #[derive(Clone, Copy, Debug)]
-pub struct GroupConfig {
+pub struct MlsGroupConfig {
```

Comment:
I'm generally not keen on using the `Mls` prefix everywhere. But at least it's consistent this way..

Reply:
I'm not attached to it, but if we want to change it, we should do it soon, before even more things depend on that API. I'll merge for now, but again, I'm not opposed to a different naming scheme.